src/components/review-board.css

```diff
@@ -0,0 +1,39 @@
+.review-list {
+  margin: 5px;
+  list-style-type: none;
+  display: grid;
+  grid-gap: 50px;
+  grid-template-columns: repeat(auto-fill, 500px);
```

Comment:
Good use of grid, and I can see you're thinking about different screen widths, but don't forget to consider how other UI elements and positions may need to change for desktop. You don't need to do this just yet, but something to think about for future tickets.